data/mods/DinoMod/monstergroups/wilderness.json

```
@@ -12,6 +12,19 @@
         "pack_size": [ 4, 8 ],
         "conditions": [ "DUSK", "DAWN", "SPRING", "SUMMER", "AUTUMN" ]
       },
+        {
+        "monster": "mon_qianzhousaurus",
+        "freq": 50,
+        "cost_multiplier": 0,
+        "pack_size": [ 4, 8 ],
+        "conditions": [ "DUSK", "DAWN", "SPRING", "SUMMER", "AUTUMN" ]
+         {
```

```suggestion
         },
         {
```
Missing a closing bracket here.